core/src/main/kotlin/org/evomaster/core/search/Individual.kt

```diff
@@ -441,7 +441,10 @@ abstract class Individual(override var trackOperator: TrackOperator? = null,
     }
 
     private fun isLocalIdsNotAssigned() : Boolean{
-        return flatView().all { it.getLocalId() == ActionComponent.NONE_ACTION_COMPONENT_ID}
+        return flatView().all { !it.hasLocalId ()
```

Comment:
instead of `all`, shouldn't it be `any`?

Reply:
also, isn't this just `! areAllLocalIdsAssigned()` ?

Reply:
it should be `all`.
the local id will be assigned only if none of its children has local id.

in this case, it would not be `! areAllLocalIdsAssigned()`

Reply:
so, if 1 out 10 has `localID`, then the function returns `true`, which could be interpreted as the localIds are initialized.
if the logic must be with `all`, maybe a better name is `areAllLocalIdsNotInitialized()`

Reply:
I will change the name.

if the structural element has 10 children, only one of them has local id. it should be a bug. All of its children should be initialized at the same time.
currently, if the one out of 10 children has local id, then the rest of the children will not be assigned with the local id, thus the problem would be identified when checking `validLocalId`.